pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "IRKsome"
+version = "0.0.1"
```

Comment:
I know it doesn't really matter, but feels like we might be justified to bump this version number...

Reply:
Do you have a suggestion as to what number we're on by now?

Reply:
If you're not committing to making versioned releases (almost certainly overkill) then I don't think this matters at all.

Reply:
Fair enough